index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a href="#" class="logo">Abimbola</a>
+        <a><i class="fa fa-bars cursor"></i></a>
+      </nav>
+    </header>
+    <section class="main-section img">
+        <div>
+          <h1 class="greetings">Hey There.</h1>
+          <h2 class="greetings">I'm Abimbola</h2>
+          <h3 class="title">I'am a Software Developer</h3>
```

Comment:
Hi @Dipeshtwis, The `span` tag was in the header tag but I would go with your own suggestion, thanks 🙂

Reply:
Use `h1` for title only. 
For the sub-title, we should use `h2` to `h5` header tag. and the reason is the same as above.